spec/ameba/cli/cmd_spec.cr

```diff
@@ -20,8 +20,14 @@ module Ameba::Cli
 
       %w(-c --config).each do |f|
         it "accepts #{f} flag" do
-          c = Cli.parse_args [f, "config.yml"]
-          c.config.should eq Path["config.yml"]
+          c = Cli.parse_args [f, "shard.yml"]
+          c.config.should eq Path["shard.yml"]
```

Comment:
I had to change the config file we pass to `parse_args` to a file which actually exists so I chose the shard file.

I could create a temp file in the test and throw it away afterwards?